iBudget/ibudget/views.py

```diff
@@ -0,0 +1,43 @@
+"""
+Views module for handling AWS S3
+"""
+from django.http import HttpResponse
+from django.views import View
+from utils.aws_helper import upload, get_image_url, del_photo, check_if_in_bucket
+from utils.response_helper import (RESPONSE_400_INVALID_DATA)
+
+
+class AwsCall(View):
```

Comment:
This is a bad name. It should not have any references fo inner implementation. Maybe something like `fileHandler`... But in general, the class is not super necessary here. You can just use functions.

Reply:
Thanks for remark!

I created a class because at the beginning you and Liubomyr recommended to do that, also I wanted to implement class-based view to check it in action.

I will change the name of the class, still is it obligatory to turn the class into functions?